### PR TITLE
Add support for SPV_KHR_compute_shader_derivative

### PR DESCRIPTION
**Pull request by EpicJeanNoeMorissette**
Replace support for SPV_NV_compute_shader_derivatives with the new KHR (many lists had the vendor extension twice, replace one with the new KHR).

### CODE REVIEW

File: source/opt/aggressive_dead_code_elim_pass.cpp [hunk 1]
```diff
@@ -1010,7 +1010,7 @@ void AggressiveDCEPass::InitExtensions() {
       "SPV_NV_bindless_texture",
       "SPV_EXT_shader_atomic_float_add",
       "SPV_EXT_fragment_shader_interlock",
-      "SPV_NV_compute_shader_derivatives",
+      "SPV_KHR_compute_shader_derivatives",
```
Comment (alan-baker) [inline source/opt/aggressive_dead_code_elim_pass.cpp:1013]: I think this should be additive. The capabilities and execution modes are aliases, but these strings are not. So in all these passes it makes sense to list both as handled.

Comment (alan-baker) [reply source/opt/aggressive_dead_code_elim_pass.cpp:1013]: Ah ok, makese sense

**Review by alan-baker — APPROVED** [review]

**Review by dneto0 — APPROVED** [review]

**Comment by alan-baker** [comment]
It looks like you missed a few test updates.

```
[  PASSED  ] 6974 tests.
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] ValidateDerivatives.OpDPdxWrongExecutionModel
[  FAILED  ] ValidateDerivatives.NoExecutionModeGLCompute
```

Try `ctest -j<N>` after fixing them to confirm those are the only failures.

**Comment by alan-baker** [comment]
Please fix the formatting.

**Comment by alan-baker** [comment]
> Please fix the formatting.

I fixed this.

**Pull request merged**